karpenter/datadog_checks/karpenter/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.0.1'
```

I think, cc @iliakur

Reply:
That's correct! Currently with 1.0.0 in about.py we won't be able to release with the `--new` flag.